plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -472,6 +472,13 @@ protected String createSchemaSql(String schemaName)
         return "CREATE SCHEMA " + schemaName + " WITH (location = 's3://" + bucketName + "/" + schemaName + "')";
     }
 
+    @Override
+    public void testCreateTableWithLongTableName()
+    {
+        // TODO: Fails with java.net.SocketTimeoutException: Read timed out at ThriftHiveMetastore.createTable
```

Comment:
That sounds like a failure on metastore side (like HTTP 500) and it didn't return any response.

Do metastore logs say anything?